### PR TITLE
Simplify `string` to Public Key / Signature / Hash conversion

### PR DESCRIPTION
**Pull request by Geod24**
See each commit for detail.

### CODE REVIEW

File: src/modules/data/Hash.ts [hunk 1]
```diff
@@ -123,9 +115,9 @@ export class Hash
      * @param endian The byte order
```
Comment (TrustHenry) [inline src/modules/data/Hash.ts]: Need to erase this phrase.

Comment (TrustHenry) [inline src/modules/data/Hash.ts]: Need to remove this phrase.

File: src/modules/data/PublicKey.ts [hunk 2]
```diff
@@ -37,17 +37,32 @@ export class PublicKey
     public static Width: number = sodium.crypto_sign_PUBLICKEYBYTES;
 
     /**
-     * Constructor
-     * @param bin The binary data of the public key
+     * Construct a new instance of this class
+     *
+     * @param data   - The string or binary representation of the public key
+     * @param endian - The byte order
```
Comment (TrustHenry) [inline src/modules/data/PublicKey.ts]: JSDoc comment also supports `-`.
Reading is good if `-` are included.
However, most documents do not have `-`.
https://jsdoc.app/tags-param.html

Comment (Geod24) [reply src/modules/data/PublicKey.ts]: Indeed, will revert

**Comment by Geod24** [comment]
Comments addressed.

**Review by TrustHenry — APPROVED** [review]
LGTM

**Pull request merged**